src/profile/__mocks__/savingEditedBio.mockStore.js

```diff
@@ -114,11 +114,11 @@ module.exports = {
         grade: '0.89',
         courseId: 'course-v1:edX+DemoX+Demo_Course',
         courseOrganization: 'edX',
-        modifiedDate: '2019-03-04T19:31:39.930255Z',
+        modifiedDate: '2019-03-05T19:31:39.930255Z',
         isPassing: true,
         downloadUrl: 'http://www.example.com/',
         certificateType: 'verified',
-        createdDate: '2019-03-04T19:31:39.896806Z'
+        createdDate: '2019-03-05T19:31:39.896806Z'
```

Comment:
Are these changes introduced by the command you ran to update snapshots?

Reply:
The date has been updated in the snapshot so we will have to update our mock data too but this is still a work in progress as the tests aren't behaving consistently across CI and on Local environment due to the datetime not being in utc format.